node.go

```diff
@@ -63,14 +63,21 @@ func PrefixForCidVersion(version int) (cid.Prefix, error) {
 	}
 }
 
+// Prefix returns the CID Prefix for this ProtoNode, it is never nil
+func (n *ProtoNode) Prefix() cid.Builder {
```

Comment:
If we're going to break this anyways, we might as well call this `CidBuilder` and `SetCidBuilder`. Any reason not to?

Reply:
No.  I think I will just use `Builder` though.